joomla/joomla-cms
# [4.0] com_content - Remove 'Links' feature from 'Images and Links'

Pull Request for Issue # .

### Summary of Changes
This PR removes the **links** from the **Images and Links** tab of com_content. Considering we now have custom fields, I fail to see the need for these options. Removal simplifies the views and reduces the number of options.

### Testing Instructions
Apply patch and ensure fields associated fields with these links have been removed (the now **Images** tab and **Options** tab)

### Documentation Changes Required
Yes
